go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/cosmos/go-bip39 v1.0.0
 	github.com/cosmos/ibc-go/v5 v5.0.1
 	github.com/davecgh/go-spew v1.1.1
-	github.com/ethereum/go-ethereum v1.10.19
+	github.com/ethereum/go-ethereum v1.10.24-0.20221109065213-7dc5e785a8fd
```

Comment:
needs update to latest go-ethereum

Reply:
it's in the main branch, do you mean update to current main, or wait for a point release.

Reply:
Do you mean update to latest go-ethereum directly will break things?

Reply:
yes, it's breaking. See #1254

Reply:
I see, maybe we close this PR and prioritize the #1254